test/test_helper.exs

```diff
@@ -1,6 +1,18 @@
 defmodule Kitto.TestHelper do
   def atomify_map(map) do
-    for {key, value} <- map, into: %{}, do: {String.to_atom(key), value}
+    for {key, value} <- map, into: %{} do
+      {if(is_atom(key), do: key, else: String.to_atom(key)), value}
```

Comment:
That doesn't work in this context. If I change that line to:

```elixir
{if is_atom(key), do: key, else: String.to_atom(key), value}
```

It generates a compile time SyntextError: `test/test_helper.exs:4: syntax error before: value`